### PR TITLE
Gpt2 model card

### PR DESCRIPTION
**Pull request by sgugger**

### CODE REVIEW

File: model_cards/bert-base-uncased-README.md [hunk 1]
```diff
@@ -1,10 +1,227 @@
...
+  predict if the two sentences were following each other or not.
+
+This way, the model learns an inner representation of the English language that can then be used to extract features
+useful for downstream tasks: if you have a dataset of labeled sentences for instance, you can train a standard
+classifier using the features produced by the BERT model as inputs.
+
+## Intended uses & limitations
+
+You can use the raw model for either masked language modeling or next sentence prediction, but it's mostly intended to
+be fine-tuned on a downstream task. See the [model hub](https://huggingface.co/models?filter=bert) to look for
```
Comment (yjernite) [inline model_cards/bert-base-uncased-README.md:44]: filter=gpt2

Comment (sgugger) [reply model_cards/bert-base-uncased-README.md:44]: Oh good catch!

**Comment by sgugger** [comment]
(@julien-c approved offline)

File: model_cards/gpt2-README.md [hunk 2]
```diff
@@ -1,10 +1,147 @@
...
 
+
+# GPT-2
+
+Pretrained model on English language using a causal language modeling (CLM) objective. It was introduced in
+[this paper](https://d4mucfpksywv.cloudfront.net/better-language-models/language_models_are_unsupervised_multitask_learners.pdf)
+and first released at [this page](https://openai.com/blog/better-language-models/).
+
+Disclaimer: The team releasing GPT-2 did not write a model card for this model so this model card has been written by
+the Hugging Face team.
```
Comment (julien-c) [inline model_cards/gpt2-README.md:17]: They _did_ write a model card in their original repo: https://github.com/openai/gpt-2/blob/master/model_card.md (cc @thomwolf)

Would be nice to merge the two if some information is missing from ours.

Comment (sgugger) [reply model_cards/gpt2-README.md:17]: Oh I did not see that, will merge the two of them and adapt the disclaimer.

**Pull request merged**